store/sqlstore/command_webhook_store.go

```diff
@@ -82,7 +107,17 @@ func (s SqlCommandWebhookStore) TryUse(id string, limit int) *model.AppError {
 func (s SqlCommandWebhookStore) Cleanup() {
 	mlog.Debug("Cleaning up command webhook store.")
 	exptime := model.GetMillis() - model.COMMAND_WEBHOOK_LIFETIME
-	if _, err := s.GetMaster().Exec("DELETE FROM CommandWebhooks WHERE CreateAt < :ExpTime", map[string]interface{}{"ExpTime": exptime}); err != nil {
+
+	query := s.getQueryBuilder().
+		Delete("CommandWebhooks").
+		Where(sq.Lt{"CreateAt": exptime})
+
+	queryString, args, err := query.ToSql()
+	if err != nil {
+		mlog.Error("Failed to build query when trying to perform a cleanup in command webhook store.")
```

Comment:
Thanks for applying that patch @joshuabezaleel!! :rocket: 

We still need the early `return` here; if building the query fails, the function should stop there instead trying to run it in the database:

```suggestion
		mlog.Error("Failed to build query when trying to perform a cleanup in command webhook store.")
		return
```

Reply:
Truly sorry that I missed that, Miguel. Thank you for pointing it up. Added :slightly_smiling_face: